.github/workflows/linting-and-tests.yml

```diff
@@ -242,8 +242,7 @@ jobs:
     strategy:
       matrix:
         grafana_version:
-          - 10.1.7
-          - 10.3.3
+          - 11.1.3
```

Comment:
nice 🔥 

should we also still test against `10.x` versions? wdyt?

Reply:
Not really possible anymore after adding these changes, afaik these UI changes are only available in grafana 11.

Reply:
Checked&tested, lowest we can do is `10.2.5` which I have it set in the e2e yaml file